contracts/axelar-operators/src/test.rs

```diff
@@ -176,3 +190,50 @@ fn fail_remove_operator_non_existant() {
     let res = client.try_remove_operator(&operator);
     assert!(res.is_err());
 }
+
+#[test]
+fn test_execute() {
+    let (env, _, client, target) = setup_env();
+
+    let owner = Address::generate(&env);
+    let operator = Address::generate(&env);
+
+    client.initialize(&owner);
+
+    let is_operator_initial = client.is_operator(&operator);
+    assert!(!is_operator_initial);
+
+    // set operator as an operator
+    client.add_operator(&operator);
+
+    // call execute as an operator
+    client.execute(
+        &operator,
+        &target,
+        &symbol_short!("method"),
+        &Vec::new(&env),
+    );
+
+    assert_emitted_event(&env, -1, &target, (symbol_short!("executed"),), ());
+}
+
+#[test]
+fn fail_execute_not_operator() {
+    let (env, _, client, target) = setup_env();
+
+    let owner = Address::generate(&env);
+    let operator = Address::generate(&env);
+
+    client.initialize(&owner);
+
+    let is_operator_initial = client.is_operator(&operator);
+    assert!(!is_operator_initial);
+
+    // set operator as an operator
+    client.add_operator(&operator);
+
+    // call execute with a non-operator, should panic
+    let res = client.try_execute(&owner, &target, &symbol_short!("method"), &Vec::new(&env));
+
+    assert!(res.is_err());
+}
```

Comment:
add a test case for when the target call fails, to ensure the whole call panics with an appropriate error